todo

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+source $HOME/.todorc
+
+TODO_PATH="${TODO_PATH:=$HOME/.todo}"
+add_task(){
+
+	temp=$(cat $TODO_PATH/id.txt)
+	ID="${temp:=1}"
+	ID=$(($ID+1))
+	echo $ID >$TODO_PATH/id.txt
+	echo $ID $1 >> $TODO_PATH/todo.txt 
```

Comment:
You need to handle the case when the user doesn't provide any input after `todo new`